.github/workflows/ci-on-pull-request-k3d.yaml

```
@@ -11,8 +11,8 @@ on:
       - "**.json"
       - "aws/**"
       - ".github/workflows/ci-on-pull-request-aws.y*ml"
-      - ".github/workflows/test-aws-package.y*ml"
-      - ".github/workflows/test-aws-upgrade.y*ml"
+      - ".github/workflows/**-aws-**.y*ml"
+      - ".github/workflows/**-aws-**.y*ml"
```

This looks like a duplicate of the line right above it. What do you think about removing the duplicate and updating this pattern to use the following?:

```suggestion
      - ".github/workflows/**aws**.y*ml"
```

The current `**-aws-**` pattern won't handle `.github/workflows/ci-on-pull-request-dubbd-aws.yaml` and `.github/workflows/ci-on-pull-request-uds-core-aws.yaml` because they do not have a trailing `-` after `aws`

---

thank you!